lib/stack/managementToken/index.js

```diff
@@ -0,0 +1,111 @@
+
+    /**
+     * @description The Delete ManagementToken call is used to delete an existing ManagementToken permanently from your Stack.
+     * @memberof ManagementToken
+     * @func delete
+     * @returns {Object} Response Object.
+     * @example
+     * import * as contentstack from '@contentstack/management'
+     * const client = contentstack.client()
+     *
+     * client.stack({ api_key: 'api_key'}).managementToken('management_token_uid').delete()
+     * .then((response) => console.log(response.notice))
+     */
+    this.delete = deleteEntity(http)
+
+    /**
+     * @description The fetch ManagementToken call fetches ManagementToken details.
+     * @memberof ManagementToken
+     * @func fetch
+     * @returns {Promise<ManagementToken.ManagementToken>} Promise for ManagementToken instance
+     * @example
+     * import * as contentstack from '@contentstack/management'
+     * const client = contentstack.client()
+     *
+     * client.stack({ api_key: 'api_key'}).managementToken('management_token_uid').fetch()
+     * .then((managementToken) => console.log(managementToken))
+     *
+     */
+    this.fetch = fetch(http, 'token')
+  } else {
+    /**
+     * @description The Create a ManagementToken call creates a new ManagementToken in a particular stack of your Contentstack account.
+     * @memberof ManagementToken
+     * @func create
+     * @returns {Promise<ManagementToken.ManagementToken>} Promise for ManagementToken instance
+     *
+     * @example
+     * import * as contentstack from '@contentstack/management'
+     * const client = contentstack.client()
+     * const token = {
+     *     name: 'Test',
+     *     description: 'This is a demo token.',
+     *     scope: [{
+     *              module: 'environment',
+     *              environments: ['development'],
+     *              acl: {
+     *                read: true
+     *              }
+     *            }]
+     * }
+     *
+     * client.stack().managementToken().create({ token })
+     * .then((managementToken) => console.log(managementToken))
+     */
+    this.create = create({ http: http })
+
+    /**
+     * @description The ‘Get all managementToken’ request returns comprehensive information about all managementToken created in a stack.
+     * @memberof ManagementToken
+     * @func query
+     * @returns {ContentstackCollection} Instance of ContentstackCollection.
+     * @example
+     * import * as contentstack from '@contentstack/management'
+     * const client = contentstack.client()
+     *
+     * client.stack().managementToken().query({ query: { name: 'token_name' } })).find()
+     * .then((contentstackCollection) => console.log(contentstackCollection))
+     */
+    this.query = query({ http: http, wrapperCollection: ManagementTokenCollection })
+  }
+}
+
+export function ManagementTokenCollection (http, data) {
+  const obj = cloneDeep(data.tokens) || []
+  const managementTokenCollection = obj.map((userdata) => {
```

Comment:
Change the callback func param name here. Giving proper naming according to the value it is receiving.

Reply:
Changed the param name.